ci/workflows/pull-request.yml

```
@@ -333,6 +333,25 @@ steps:
     timeout_in_minutes: 10
     retry: *auto-retry
 
+  - label: "build-image-on-pr"
+    key: "build-image-on-pr"
+    if: build.pull_request.labels includes "build-image-on-pr" || build.env("CI_STEPS") =~ /(^|,)build-image-on-pr(,|$$)/
```

```suggestion
    if: build.pull_request.labels includes "ci/pr/build-docker-image" || build.env("CI_STEPS") =~ /(^|,)build-image-on-pr(,|$$)/
```

So the naming convention is consistent.